packages/react/package.json

```diff
@@ -18,10 +18,11 @@
   "scripts": {
     "dev": "vite",
     "build": "npm-run-all build:ts build:vite",
-    "build:analyze": "cross-env ANALYZE_BUNDLE=true vite build",
+    "build:analyze": "vite build",
```

Comment:
Is die `build:analyze` nog wel nodig? Het commando is nu hetzelfde als `build:vite`